README.md

```diff
@@ -153,27 +155,94 @@ The notebook architectures are as follows:
+  - [https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf)
+- __rsem__: 
+  - [https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323)
+- __star fusion__: 
+  - [https://github.com/STAR-Fusion/STAR-Fusion/wiki](https://github.com/STAR-Fusion/STAR-Fusion/wiki)
+  - [http://biorxiv.org/content/early/2017/03/24/120295](http://biorxiv.org/content/early/2017/03/24/120295)
+- __mutect__: 
+  - [https://software.broadinstitute.org/cancer/cga/mutect](https://software.broadinstitute.org/cancer/cga/mutect)
+  - [https://youtu.be/rN-cLrb5aGs](https://youtu.be/rN-cLrb5aGs)
+  - [https://gatk.broadinstitute.org/hc/en-us/articles/360036490432-Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360036490432-Mutect2)
+  - [https://www.nature.com/articles/nbt.2514](https://www.nature.com/articles/nbt.2514)
+- __gatk cnv__:
+  - [https://software.broadinstitute.org/gatk/documentation/article?id=11682](https://software.broadinstitute.org/gatk/documentation/article?id=11682)
+- __strelka__:
+  - [https://www.nature.com/articles/s41592-018-0051-x](https://www.nature.com/articles/s41592-018-0051-x)
+  - [https://github.com/Illumina/strelka](https://github.com/Illumina/strelka)
+
+The following flowchart provides another good overview of what the pipeline is doing.
+
+![](https://github.com/broadinstitute/ccle_processing/blob/editREADME/documentation/updated-flowchart.png)
+
+_What is explained below comes from the notebook's documentations and might be better understood by reading them directly on the notebooks_
+
+
+### 1. Uploading and Preprocessing <a name="upload-preprocess"></a>
 
 The first phase really is about getting samples generated at the broad and located into different places. Looking for duplicates and finding/adding the metadata we will need in order to have coherent and complete sample information. __This is not something that you would need to run. you can skip directly to part2__.
 
 **Remarks:** 
 - in the initialization you might want to remove any import related to `taiga` and `gsheet` to not cause any errors.
 - feel free to reuse `createDatasetWithNewCellLines`, `GetNewCellLinesFromWorkspaces` or any other function for your own needs.
 
-### 2. Running Terra Pipelines
+### 2. Running Terra Pipelines <a name="running-terra-pipelines"></a>
+
+We are using Dalmatian to send requests to Terra, so before running this part, you need to make sure that your dalmatian `WorkspaceManager` object is initialized with the right workspace you created and that the functions take as input you workflow names. You also need to make sure that you created your sample set with all your samples and that you initialized the `sampleset` string with its name.
+You can then run the following two pipelines on your samples. The whole process may take 3-7 days.
+
+#### Copy Numbers
```

Comment:
we should not call it this way. it does CN and somatic mutations (and actually germline mutations and SV)